cmd/terramate/cli/cloud.go

```diff
@@ -798,7 +798,8 @@ func setDefaultGitMetadata(md *cloud.DeploymentMetadata, commit *git.CommitMetad
 }
 
 func setGithubActionsMetadata(md *cloud.DeploymentMetadata) {
-	md.GithubActionsDeploymentTriggeredBy = os.Getenv("GITHUB_ACTOR")
+	md.GithubActionsDeploymentActor = os.Getenv("GITHUB_ACTOR")
+	md.GithubActionsDeploymentTriggeredBy = os.Getenv("GITHUB_TRIGGERING_ACTOR")
```

Comment:
I don't see this documented [here](https://docs.github.com/en/actions/learn-github-actions/variables). Do you have any source for this or is it pure experimental?

Reply:
Oh [found it](https://docs.github.com/en/actions/learn-github-actions/contexts) `github.triggering_actor`